storm-core/src/clj/org/apache/storm/daemon/logviewer.clj

```diff
@@ -311,7 +311,7 @@
 
 (defn get-log-user-group-whitelist [fname]
   (let [wl-file (ServerConfigUtils/getLogMetaDataFile fname)
-        m (clojurify-structure (Utils/readYamlFile wl-file))]
+        m (clojurify-structure (Utils/readYamlFile (.toString wl-file)))]
```

Comment:
Maybe better to use `getAbsolutePath` or `getCanonicalPath` to clearly represent path. I used `getAbsolutePath` from #2204 but I can change it if others claim that `getCanonicalPath` is more preferred.